[Release Guide] Document Test Days (#739)
submariner-io/submariner-website: src/content/development/contribution-guide/_index.en.md

### Test Days

Test days are held in order to validate pre-released versions by the project members and the wider community.
Any bugs opened on a test day will be triaged soon after.

> The goals of these days are:

Reviewer: ```suggestion
The goals of these days are to:
```

Reviewer 2: IMHO it's okay as-is

* Verify any new features that were introduced during the last [sprint](#sprints).

> * Validate any [bugs](#bugs) that were fixed were actually fixed.

Reviewer 2: ```suggestion
* Validate any [bugs](#bugs) that were closed were actually fixed.
```

* Find any regressions in existing functionality.

Test days will be led by one of the project members, who will be responsible for:

> * Creating a test day document, if one doesn't exist yet.

Reviewer: ```suggestion
* Creating a test day spreadsheet, if one doesn't exist yet by Using the [test day template].
```

>   * Using the [test day template].

Reviewer: Merged with above point.
